src/analyses/libraryFunctions.ml

```diff
@@ -292,12 +297,33 @@ let posix_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("getaddrinfo", unknown [drop "node" [r]; drop "service" [r]; drop "hints" [r_deep]; drop "res" [w]]); (* only write res non-deep because it doesn't write to existing fields of res *)
     ("fnmatch", unknown [drop "pattern" [r]; drop "string" [r]; drop "flags" []]);
     ("realpath", unknown [drop "path" [r]; drop "resolved_path" [w]]);
+    ("memccpy", special [__ "dest" [w]; __ "src" [r]; drop "c" []; drop "n" []] @@ fun dest src -> Memcpy {dest; src});
```

Comment:
This exposes a shortcoming that we should handle: We do not handle the size argument of `Memcpy`. Also I am a bit worried about classing this as `memcpy` even if that is fixed, as it simply hides the complications here.

Reply:
I created #1177 to track this.

Reply:
I'll move it to the C group, although we're decades from C23 at this point.